api/openapi.generated.yml

```diff
@@ -932,6 +932,8 @@ components:
       properties:
         query:
           type: string
+          minLength: 1
+          maxLength: 100
```

Comment:
👍

Reply:
Just curious / to have context: Was `100` chosen for the limit by any specific criteria? Could we easily adjust it in the future if we discover folx are entering really long term strings?

Reply:
Doesn't seem hard to update . If the frontend also has `<input min={1} max={100}` - we have to keep them in sync

Reply:
But there's no technical/performance/etc limit we need to be mindful of? 100 is an arbitrary choice?

Reply:
According to @chouinar comment here it appears arbitrary: https://betagrantsgov.slack.com/archives/C05TSL64VUH/p1711038834496529?thread_ts=1710957436.730929&cid=C05TSL64VUH

Reply:
> The limit for your search box should be as long as possible, and no longer. For example, if your search engine only supports 256 characters, then don't allow users to enter in 257 characters. You should not make your text limit any shorter than that supported by the engine, because there will always be a small, but possibly significant, number of users that need to search for really long phrases. You should not make it any longer than supported by the search engine, because it will result in unnecessary error messages or unexpected search results.
> — https://ux.stackexchange.com/questions/112927/character-limit-for-the-search-box-best-pratices

Trying to image how this edge case might happen, I can think of ways: 
- Someone downloaded an opportunity PDF but forgot the URL; they search for a long string copied from the PDF
- Someone enters a keyword, a title, an ID, a assistance listing number, etc; the whole lot adds up quickly

Reply:
Adjusting this value takes 2 seconds (change the number in the API schema + a test).

We can make it longer, but of the two scenarios you put there, I think only the first would be valid at the moment. Supplying a keyword AND title AND id will almost certainly never return results as implemented. If you entered `HHS health 00.123` that exact order of words would need to appear in at least one field we check against to return results, we aren't tokenizing and searching each individually.